### ISSUE TITLE
Dialogue: Focus on block content when created

### ISSUE THREAD

**Issue by retrofox** [Automattic/jetpack#18228]
#### Steps to reproduce the issue

1. Create/edit a Conversation block.
2. Into it, create a new dialogue block

#### What I expected
The block content component should be focused, showing the caret right there.

#### What happened instead
The block is selected, but no making focus on the block content.

**Screenshots**
expect | happen
--------|--------
![image](https://user-images.githubusercontent.com/77539/103892108-c0e7d700-50c9-11eb-8f57-33cc4974fd61.png) | ![image](https://user-images.githubusercontent.com/77539/103892217-f55b9300-50c9-11eb-9564-a48e36e021a6.png)
